CIScripts/Tests/AgentTests.ps1

```diff
@@ -27,6 +48,7 @@ function Test-Agent {
         $env:Path += ";C:\Program Files\Juniper Networks\Agent"
     }
     Initialize-TestConfiguration -Session $Session -TestConfiguration $TestConfiguration
+    Invoke-Command -Session $Session -ScriptBlock $DefineTestIfAnOutputSuggestsThatAllTestsHavePassed
```

Comment:
I'm not sure I understand why we define the function in this script block and not just in global scope, could you explain why?

Reply:
That's because functions defined in (local) global scope are not available in remote sessions.

Reply:
As I know, it is the only way to 'transfer' function definition to remote session.